sys-devel/slibtool/slibtool-0.5.31.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A skinny libtool implementation, written in C"
+HOMEPAGE="https://git.midipix.org/cgit.cgi/slibtool"
+SRC_URI="https://git.midipix.org/cgit.cgi/${PN}/snapshot/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm"
+
+src_configure() {
+	./configure --host=${CHOST} --prefix="${EPREFIX}"/usr || die
```

Comment:
Have this been asked before, but why wouldn't `econf` work here? Please add a comment above this line explaining why, if it doesn't.

Rest looks good.